### ISSUE TITLE
Inbuilt Interlaced Stereo 3D Option - Enable 'X-Axis' for Portrait Mode Orientation

### ISSUE THREAD

**Issue by Rud8289** [vpinball/vpinball#690]
Hello, I have a passive (interlaced) 3D TV in my cabinet.  The TV (LG) logo is located against the left rail of the cabinet. My system is in portrait mode (all tables rotated to 0).  I'm on the latest GL and non-GL (DX9) 10.8 builds.  This is a request for both GL and non-GL versions (DX9) of 10.8 if possible. 

I was hoping that I could enable interlaced 3D to run 'vertically' on my TV (would technically be on the TV's X-Axis), but, no matter the setting, the interlacing always runs on the TV's Y-Axis when in Portrait Mode.  I was hoping to have an option that rotates the interlacing 90° when in portrait mode. 

To enable this vertical interlacing option for portrait screen orientation in Future Pinball (BAM), the option is called: Passive interlaced-vertical split. And in Reshade (SuperDepth3D) it’s called: Column Interlaced

![thumbnail1](https://github.com/vpinball/vpinball/assets/112511853/a86b75b9-5183-4560-a91b-8a3c601d4c38)
![thumbnail2](https://github.com/vpinball/vpinball/assets/112511853/2675526d-6592-4674-884b-c2eaea6448f8)

**Comment by gjupp** [comment]
This appears to be a bug rather than a feature request as toggling the "Use Y Axis" switch seems to have no effect even when windows is set to portrait mode, with relation to either of the Interlaced LG 3D TV settings. It is always stuck in "Use Y Axis" mode regardless of whether it is ticked on unticked.

In addition, I'm also seeing that if either of the Interlaced LG 3D TV modes is selected, the POV becomes distorted and the table length is greatly reduced (almost reduced to 50%) and POV needs to be readjusted to accommodate. Top and Bottom and Side by Side also have issues with POV being greatly affected. Are you seeing this same additional problem @Rud8289 (I'm currently on DX version, revision 1348)

**Comment by Rud8289** [comment]
Hi gjupp (Gravy), yes I guess you’re right.  It’s technically a bug.  I wrote it as a feature request since I can overcome it by running my setup in landscape (table orientation 270, as 99% of VPX users run) and the interlacing runs vertically… so portrait mode would just be an accommodation for me. 

Additionally the DX9 version seems to have more 3D ‘tuning’ option over the GL version. I’d love for this to be fixed in for the GL version as I’m trying to overcome the plastic (ramp) rendering issue in 3D (more below).  

I don’t have issues with POV since I’m running a Kinect v2 (BAM Headtracking), but… I’m not getting much depth when I run interlaced + landscape (both GL and DX9).  I just read some comments between you and Vincent within another closed issue thread a while back.  Looks like he’s aware of this issue and might look into it for v10.9 or v11.  Hoping that’s the case.  I’m using Kinect + VPX + 4k 3D TV + Reshade (SuperDepth3D) and it’s awesome (still some screen door effect though unfortunately, due to how close I am to the screen, but definitely a major improvement over 1080 3D), but was hoping I could utilize VPX’s inbuilt 3D to 1. solve the plastics issue (i.e. playfield rendered into the bottom of plastic ramps) 2. Achieve some performance gains 3. Have a better overall 3D experience (sharper image, cleaner edges, etc.) as I’ve read the inbuilt VPX might take care of.

**Comment by gjupp** [comment]
You are lucky having a 4K LG Passive 3D screen. What model number is it? I assume its 50 inch or something?

Would like to chat with you more on the topic, assuming you are on VPF, what is your nickname there? I'm Gravy on VPF which you already guessed.

**Comment by Rud8289** [comment]
Yeah it’s a 49” LG 49UB8500.  Got lucky and found one second hand a couple of years ago.  It’s pretty big, but it is what it is.  I built a custom cab for it. And yes, I’m on VPF.  I’m: Rund8289. Feel free to send a message.

**Comment by Rud8289** [comment]
Just wanted to add:  To enable this vertical interlacing option for portrait screen orientation in Future Pinball (BAM), it’s called: Passive interlaced-vertical split.  And in Reshade (SuperDepth3D) it’s called: Column Interlaced

**Comment by gjupp** [comment]
Can you change the label on this and mark it as a bug please or else it may not get noticed.

There are no consumer LG 3D screens that I know of (or other 3D screens using polarised glasses) that use a vertical polarised interlacing filter. As such it is incorrect for the software to rotate the direction of the interlacing comb orientation when a user is  has their windows set to portrait orientation (resulting in this inability to use a 3D polarised screen in portrait setting).

**Comment by Rud8289** [comment]
I see what you mean.  Good idea, I'll have to close this issue and open a new issue labeled as 'bug', as I'm not able to edit and re-label.

**Comment by Rud8289** [comment]
Closing this issue and opened a new issue labeled as 'bug', as I'm not able to edit and re-label.
